src/Corporation/ui/modals/SellMaterialModal.tsx

```diff
@@ -3,11 +3,11 @@ import { dialogBoxCreate } from "../../../ui/React/DialogBox";
 import { Material } from "../../Material";
 import { SellMaterial } from "../../Actions";
 import { Modal } from "../../../ui/React/Modal";
-import Typography from "@mui/material/Typography";
 import TextField from "@mui/material/TextField";
 import Button from "@mui/material/Button";
 import { KEY } from "../../../utils/helpers/keyCodes";
-
+import { Typography, FormControlLabel, Switch, Tooltip } from "@mui/material";
+import { Division } from "src/Corporation/Division";
```

Comment:
So this doesn't end up causing any issues because it's being imported just as a type, but we actually need to use relative paths for imports. If Division was being imported for runtime use then you would get a build error due to this being an absolute path.

Also when importing a class just to reference its type, it's best practice to do `import type` and put it above all the regular imports, this makes it clearer what the import dependencies are as far as figuring out circular dependencies. Types can have circular imports and they're fine with that, but runtime imports can sometimes cause issues when there is a circular dependency.

Reply:
If you use VSCode you can have it automatically do relative paths for things it autosuggests, I didn't know about this for the longest time and was manually converting to relative paths:

File -> Preferences -> Settings -> User or Workspace tab (I have it set for User but Workspace is more correct) -> Filter for "import" -> Select Extensions/Typescript -> set "Import module specifier" for JS and TS to "relative"

![image](https://github.com/bitburner-official/bitburner-src/assets/84951833/1da090f0-b1b6-4381-b148-ca9401522901)

Reply:
ill keep that in mind thanks!